packages/frontend/src/pages/Space.tsx

```diff
@@ -44,7 +45,10 @@ const Space: FC = () => {
     const { projectUuid, spaceUuid } = useParams<{
         projectUuid: string;
         spaceUuid: string;
-    }>();
+    }>() as {
+        projectUuid: string;
+        spaceUuid: string;
+    };
```

Comment:
I like this, wonder if we should have a reusable hook like `useProjectParams` that does not return `undefined` so we can avoid the non-null assertion operator